ArcadiaParties-SPA/src/app/app.component.html

```diff
@@ -1,3 +1,2 @@
-<h1>WELCOME</h1>
-<h1> Title is: {{ title$ | async }} </h1>
- 
\ No newline at end of file
+<app-header></app-header>
```

Comment:
I meant it a bit different way.

Reply:
This way?
![image](https://user-images.githubusercontent.com/55158384/75106272-acbef680-562b-11ea-8ee6-e2dea8635724.png)

Reply:
Yes.